apps/edxec/templates/services/app/job_00_collectstatic.yml.j2

```diff
@@ -0,0 +1,79 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: edxec-collectstatic-{{ job_stamp }}
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxec
+    service: edxec
+    version: "{{ edxec_image_tag }}"
+    job_stamp: "{{ job_stamp }}"
+    # Jobs with the "pre" job type will be executed prior to deployments
+    job_type: "pre"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  template:
+    metadata:
+      name: edxec-collectstatic-{{ job_stamp }}
+      labels:
+        app: edxec
+        service: edxec
+        version: "{{ edxec_image_tag }}"
+        job_stamp: "{{ job_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      containers:
+        - name: edxec-collectstatic-{{ job_stamp }}
+          image: "{{ edxec_image_name }}:{{ edxec_image_tag }}"
+          imagePullPolicy: Always
+          # Describe environment explicitly
+          env:
+            - name: ECOMMERCE_CFG
+              value: /usr/local/etc/ecommerce/local.yml
+            - name: DJANGO_SETTINGS_MODULE
+              value: ecommerce.settings.production
+          command:
+            - "bash"
+            - "-c"
+            - python manage.py collectstatic --noinput && python manage.py compress
+          volumeMounts:
+            - name: edxec-local
+              mountPath: /usr/local/etc/ecommerce
+            - name: edxec-v-media
```

Comment:
You shouldn't need to mount media to collectstatic?

Reply:
Good point, thx!